sharding-jdbc-example/sharding-example/sharding-raw-jdbc-example/src/main/java/org/apache/shardingsphere/example/sharding/raw/jdbc/YamlConfigurationExample.java

```diff
@@ -45,9 +47,17 @@ public static void main(final String[] args) throws SQLException, IOException {
         commonService.initEnvironment();
         commonService.processSuccess();
         commonService.cleanEnvironment();
+        CommonService countryService = getCountryService(dataSource);
```

Comment:
we should only use countryService here, commonService can be removed.

Reply:
ok